types/file-type/index.d.ts

```diff
@@ -1,16 +1,17 @@
-// Type definitions for file-type
+// Type definitions for file-type 5.2
 // Project: https://github.com/sindresorhus/file-type
 // Definitions by: KIM Jaesuck a.k.a. gim tcaesvk <http://github.com/tcaesvk/>
+//                 BendingBender <http://github.com/BendingBender>
 // Definitions: https://github.com/DefinitelyType/DefinitelyTyped
```

Comment:
I noticed the URL should be: https://github.com/DefinitelyTyped/DefinitelyTyped (added 'd' before the last slash).